internal/testhelpers/suite/suite.go

```diff
@@ -0,0 +1,50 @@
+package suite
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/stretchr/testify/require"
+	testify "github.com/stretchr/testify/suite"
+)
+
+type tSuite interface {
```

Comment:
Might be worth adding some sort of method to identify this is OUR version of the suite. That would help avoid accidentally passing the wrong suite. Eg. `IsHelperSuite()`